src/structures/paging/page_table.rs

```diff
@@ -19,18 +19,26 @@ pub enum FrameError {
 }
 
 /// A 64-bit page table entry.
-#[derive(Clone)]
+#[derive(Clone, Copy)]
```

Comment:
I don't think that it's a good idea to make PageTableEntry Copy because it essentially contains a reference.

Reply:
hmm, no idea, how I would implement then
```
    pub fn new() -> Self {
        PageTable {
            entries: [PageTableEntry::new(); ENTRY_COUNT],
        }
    }
```

Reply:
We can add an optional dependency on `array_init` again. See https://github.com/rust-osdev/x86_64/pull/116.

Reply:
fixed